float32 position_goal
int16 speed_goal
int16 accel_goal
int32 control_mode
